src/wpantund/NCPInstanceBase-Addresses.cpp

```diff
@@ -1076,68 +1085,88 @@ bool
 NCPInstanceBase::should_add_route_on_interface(const IPv6Prefix &route, uint32_t &metric)
 {
 	bool should_add = false;
-	bool route_added_by_device = false;
-	bool route_added_by_others = false;
-	RoutePreference preference_device = NCPControlInterface::ROUTE_LOW_PREFRENCE;
-	RoutePreference preference_others = NCPControlInterface::ROUTE_LOW_PREFRENCE;
 
-	std::multimap<IPv6Prefix, OffMeshRouteEntry>::iterator iter, sub_iter, upper_iter;
+	if (mAutoAddOffMeshRoutesOnInterface) {
```

Comment:
This is just moving all the off-mesh route related code under a `if (mAutoAddOffMeshRoutesOnInterface)` check.
Diff does look much simpler and more readable with "hide white-space changes".